modules/oe_content_entity/modules/oe_content_entity_organisation/README.md

```diff
@@ -2,14 +2,4 @@
 
 The Organisation entity type is used when collecting information about different types of organisations.
 
-In order for users to view, edit or delete organisations you need to explicitly grant them the right permissions.
-
-The module provides the following permissions:
-
-- `Organisation: Access canonical page`
-- `Organisation: Access overview page`
-- `Organisation: View any published entity`
-- `Organisation: View any unpublished entity`
-
-Grant the `Organisation: View any published entity` permission to the anonymous user role in order to allow your 
-site's visitors to view organisation entities.
+This module also provides the `Stakeholder` bundle to manage to create and reference project coordinators and participants.
```

Comment:
Why remove the permissions comment? if its about the contact bundles they where already removed here https://github.com/openeuropa/oe_content/commit/fb4942dc22c71e50b808a4ed4fde545193fdf3e5#diff-6630356b7b36250113c966a3913c7484

Reply:
You're right I reverted.

Reply:
Let's not bundle too much this with the project content type, I'd stay more generic, like:

```
This module also provides the `Stakeholder` bundle to be used when dealing with stakeholder organizations (for example on projects).
```

Reply:
done